### DISCUSSION TITLE
RFC: coding blogs

### DISCUSSION THREAD

**Discussion by jocmp** [jocmp/capyreader#336] [Announcements]
I want to make formatting for coding blogs better. Right now code blocks and inline code tags are too small relative to the text around them. 

But I need your help to verify formatting changes look better. If you have any programming or coding related blogs that display code in their posts please share the RSS links below. Thanks!

**Comment by Adikso** [comment]
https://secret.club/feed.xml
https://blog.trailofbits.com/feed/

Not exactly "coding blogs", but you will find code here

**Reply by jocmp** [reply]
thanks!

**Comment by jocmp** [comment]
I've added better formatting in the latest version. I'll keep this discussion open for any future feedback.
